bootstrap/_deb_common.sh

```diff
@@ -9,7 +9,11 @@
 #     - 6.0.10 "squeeze" (x64)
 #     - 7.8 "wheezy" (x64)
 #     - 8.0 "jessie" (x64)
+#   - Raspbian:
+#     - 7.8 (armhf)
 
+apt-get update
+apt-get install -y lsb-release
```

Comment:
could you please put `apt-get install ...` as a first line of `newer()` and use `--no-install-recommends`?